test/types/string/psahabu/perf/stringSearch.perfexecopts

```diff
@@ -0,0 +1 @@
+--bogus=true # search.perfkeys
```

Comment:
Is an additional flag necessary?  I though it was sufficient to have a single space followed by the `# search.perfkeys`

Reply:
Unfortunately that wasn't working :(

Reply:
Nvm, I was missing the space. I'll fix it.